### ISSUE TITLE
Upload image single/double quote issue

### ISSUE THREAD

**Issue by megamuf** [GetmeUK/ContentTools#323]
While implementin imaqge upload we ran into a very strange behaviour - on some computers (In our case all of them were laptops) the backgroun image wouldn't show up. By inspection the code we see that:
```
<div class="some specific classes" style='background-image:url("path/to/image")' />
```

if we manually revert the usage  of double/single quotes (in inspector) then the image is rendered OK.

The strangest is that this behaviour remains in all browsers and all computers have windows 10 installed.

- It works well on desktop computers.
- It fails on laptops.

How can we fix this strange behaviour?

P.S. Here is our imageuploader.fileready code (almos as in the example)

```javascript

dialog.addEventListener('imageuploader.fileready', function (ev) {
        $('.ct-dialog__close').unbind('click');
        // Upload a file to the server
        var formData;
        var file = ev.detail().file;
        //console.log(file)

        // Define functions to handle upload progress and completion
        xhrProgress = function (ev) {
            // Set the progress for the upload
            dialog.progress((ev.loaded / ev.total) * 100);
        };

        xhrComplete = function (ev) {
            var response;

            // Check the request is complete
            if (ev.target.readyState != 4) {

                return;
            }

            // Clear the request
            xhr = null;
            xhrProgress = null;
            xhrComplete = null;

            // Handle the result of the upload
            if (parseInt(ev.target.status) == 200) {
                // Unpack the response (from JSON)
                response = JSON.parse(ev.target.responseText);
                imageResponse = response;
                //console.log(response);
                //response = JSON.parse(ev.target.responseText);
                //console.log(response)

                // Store the image details
                image = {
                    size: [response.width, response.height],
                    url: response.url
                };

                // Populate the dialog
                dialog.populate(image.url, image.size);
                $('.ct-dialog__close').on('click', function () {
                    $('.ct-control--clear').trigger('click');
                });

            } else {
                // The request failed, notify the user
                new ContentTools.FlashUI('no');
                $('.ct-control--cancel').on('click', function () {
                    $('.ct-dialog__close').trigger('click');
                });

            }
        };
```

**Comment by anthonyjb** [comment]
I'm wondering if this is because we don't wrap thebackground URL in quotes in the JS, for example if you patch this lines in your CT JS https://github.com/GetmeUK/ContentTools/blob/master/build/content-tools.js#L4057 to be:

```JavaScript
style += 'background-image:url("' + this._attributes['src'] + '");';
```

Does that resolve the issue? If so please let me know and I'll patch the library.

**Comment by megamuf** [comment]
@anthonyjb
Yes, this resolve the issue!
Thank you!!!

**Comment by anthonyjb** [comment]
@megamuf thanks for letting me know I'll add this as a patch in the next release.

**Comment by anthonyjb** [comment]
Fixed in [release 1.3.1](https://github.com/GetmeUK/ContentTools/releases/tag/1.3.1).
